api/migrations/20181005143053-users-notifications-useridandnotificationid-index.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(r, conn) {
+  return r
+    .table('usersNotifications')
+    .indexCreate('userIdAndNotificationId', [
```

Comment:
This is fine, but at some point we really need to start scrubbing old notifications. These indexes + the sizes of the original tables are going to blow up our storage volumes. Not too big an issue since it's cheap, but will definitely be something to work on in the future.